### ISSUE TITLE
Nested CSS @apply mixins are breaking when minified.

### ISSUE THREAD

**Issue by Norman-Amy** [jakubpawlowicz/clean-css#939]
I've run into a problem after CSS has been minified (through [kangax/html-minifier](https://github.com/kangax/html-minifier) which uses clean-css) where the last property being applied to a CSS mixin variable loses its semi-colon. If this property is another mixin, it breaks, particularly for IE11 and Edge. Could this please be fixed so that the semi-colons are preserved?

As a side note, I've found a way around this by adding another property, but it gets messy, so if this could be fixed, it would be greatly appreciated.

### Input CSS

```css
--my-mixin: {
    @apply --another-mixin;
};
```

### Actual output CSS

```css
--my-mixin:{@apply --another-mixin};
```

### Expected output CSS

```css
--my-mixin:{@apply --another-mixin;};
```

**Comment by jakubpawlowicz** [comment]
I wasn't aware IE11 and Edge need that semicolon in `@apply` - fix is coming!

Btw, are you on latest html-minifier >= 3.3 (so using clean-css 4.x) or older?

**Comment by Norman-Amy** [comment]
Thanks for looking into the bug! I failed to mention gulp-htmlmin which I'm pinned to the latest for (3.0.0). It pulls in html-minifier v3.2.3 so my app is pinned to clean-css v3.4.25. I'll work on updating that...

**Comment by jakubpawlowicz** [comment]
@Norman-Amy quick question - how do you use `@apply` in IE11 or Edge? Is it via some kind of wrapper or Polymer? As far as I can see it's not supported natively - http://caniuse.com/#feat=css-apply-rule

**Comment by Norman-Amy** [comment]
I'm using it in Polymer web components. I believe Polymer provides a shim for browsers when it isn't supported natively.

**Comment by jakubpawlowicz** [comment]
It's interesting then that the shim can't handle `@apply` without trailing semicolon in IE11 and Edge. But I'll fix it in clean-css in 3.4 and 4.x branches.

**Comment by jakubpawlowicz** [comment]
The fix is out in 3.4.26 and 4.1.2.
